app/settings.py

```diff
@@ -22,37 +22,39 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-2r#aw*gd*!)%y=#t(*8x-7q1*yxsiifigwyrb3l5kzf)cd#3&r'
+SECRET_KEY = "django-insecure-2r#aw*gd*!)%y=#t(*8x-7q1*yxsiifigwyrb3l5kzf)cd#3&r"
```

Comment:
Should this not be put into an Environment variable on Github or Server?

Reply:
Ah yeah, it definitely should. Let's handle this separately.